pkg/cmd/pr/view/view.go

```diff
@@ -137,6 +138,8 @@ func printRawPrPreview(io *iostreams.IOStreams, pr *api.PullRequest) error {
 	fmt.Fprintf(out, "milestone:\t%s\n", pr.Milestone.Title)
 	fmt.Fprintf(out, "number:\t%d\n", pr.Number)
 	fmt.Fprintf(out, "url:\t%s\n", pr.URL)
+	fmt.Fprintf(out, "additions:\t+%s\n", cs.Green(strconv.Itoa(pr.Additions)))
```

Comment:
Since the idea with the raw output is for easier `grep`ing, I don't think that we need to prepend these numbers with `+` and `-` here. Let's drop that in this output since the `additions` and `deletions` keys are sufficient to identify those numbers.